tests/src/haxepunk/BaseTest.hx

```diff
@@ -0,0 +1,28 @@
+package haxepunk;
+
+import massive.munit.Assert;
+
+class BaseTest
```

Comment:
Instead of using inheritance, what about providing these as static methods similar to how Assert currently works?